src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -166,29 +164,21 @@ public void testLogOutput() throws Exception {
         final String secondErrorMessage = getCheckMessage(MSG_KEY, "anotherKey");
 
         verify(checker, propertyFiles, ImmutableMap.of(
-            getPath(""), Collections.singletonList(line + firstErrorMessage),
-            translationProps, Collections.singletonList(line + secondErrorMessage)));
-
-        final String osName = System.getProperty("os.name").toLowerCase(ENGLISH);
-        String expectedLogOutput;
-
-        // till https://github.com/checkstyle/checkstyle/issues/5103
-        if (osName.startsWith("windows")) {
-            expectedLogOutput = readFile("OutputTranslationCheckWindows.xml");
-        }
-        else if (osName.startsWith("linux")) {
-            expectedLogOutput = readFile("OutputTranslationCheckLinux.xml");
-        }
-        else {
-            expectedLogOutput = readFile("OutputTranslationCheckMacOS.xml");
-        }
-
-        expectedLogOutput = expectedLogOutput.replace("path_to_file", getPackageAbsolutePath());
-        expectedLogOutput = String.format(Locale.getDefault(), expectedLogOutput,
-                firstErrorMessage, secondErrorMessage).replace("'", "&apos;");
```

Comment:
> attribute value for 'message' should match: /#document/checkstyle/file/error/ expected:<Properties[ file 'InputTranslationCheckFireErrors_ja.properties' is missing].> but was:<Properties[-Datei 'InputTranslationCheckFireErrors_ja.properties' fehlt].

I forgot about this. XmlLogger doesn't have this issue since it controls what messages it prints.
@romani Is it ok if I just use `Locale.ROOT` or should I make a replace routine like the original test had?

Reply:
it is better to avoid ROOT.